src/data-mgt/python/jobs/airqo_device_measurements.py

```diff
@@ -127,12 +127,11 @@ def transform_chunk(chunk):
     # create a dataframe to hold the chunk
     data = pd.DataFrame(chunk)
 
-    # create a to hold all threads
-    threads = []
-
     # loop through the devices in the chunk
     for index, row in data.iterrows():
 
+        device_name = row['device']
```

Comment:
can use `row.get`  with a default value to avoid any exception

Reply:
Thanks @ja-odur